`test_segarray_read` failing in `make test-proto` for multi-locale

`test_segarray_read` failing in `make test-proto` for multi-locale

```
FAILED PROTO_tests/tests/io_test.py::TestParquet::test_segarray_read[None] - KeyError: 'ListCol'
FAILED PROTO_tests/tests/io_test.py::TestParquet::test_segarray_read[snappy] - KeyError: 'ListCol'
FAILED PROTO_tests/tests/io_test.py::TestParquet::test_segarray_read[gzip] - KeyError: 'ListCol'
FAILED PROTO_tests/tests/io_test.py::TestParquet::test_segarray_read[brotli] - KeyError: 'ListCol'
FAILED PROTO_tests/tests/io_test.py::TestParquet::test_segarray_read[zstd] - KeyError: 'ListCol'
FAILED PROTO_tests/tests/io_test.py::TestParquet::test_segarray_read[lz4] - KeyError: 'ListCol'
```

--- a/PROTO_tests/tests/io_test.py
+++ b/PROTO_tests/tests/io_test.py
@@ -405,7 +405,7 @@ class TestParquet:
                             for li in lists
                         ]
                     )
-                    concat_segarr.to_parquet(file_name, "test-dset", compression=comp)
+                    concat_segarr.to_parquet(file_name, "ListCol", compression=comp)
                 else:
                     # when single locale artifically create multiple files
                     for i in range(NUM_FILES):

